utils/form_validation.php

```diff
@@ -489,4 +557,139 @@ function set_target_dir(string $directory)
         $this->target_dir = $directory;
         return $this;
     }
-}
\ No newline at end of file
+}
+
+class EmailField extends Field
```

Comment:
This should imo extend StringField

Reply:
Feel free to disregard if you think this is too much work, I can maybe look at this later and refactor